exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+
+
+def read_file_move_to_dict(file_path):
+    """Read file and move atm information to dictionary.
+
+    Args:
+        file_path (string): path to atm information file.
+
+    Return:
+        A dictionary, the key is each users id.
+        the value is a list of user's password and balance.
+    """
+    try:
+        atm_options_dict = {}
```

Comment:
We don't use the type in the variable name. You can just call it atm_options.

Look over your code and change the name of all of your variable names which contain types (pycharm's refactor function should make it easier).